ncm-opennebula/src/main/pan/components/opennebula/schema.pan

```diff
@@ -532,6 +552,7 @@ datastores, vnets, hosts names, etc
 type component_opennebula = {
     include structure_component
     'datastores'    ? opennebula_datastore[1..]
+    'groups'        ? opennebula_group[]
```

Comment:
@stdweird I was checking what should be changed to implement this, and we should change tt files, unit tests and also de component code itself so I propose to include this change in a different PR and keep these lists for now.. I have created a new bug to fix this:
https://github.com/quattor/configuration-modules-core/issues/834
I will change this as soon as possible